src/RestControllers/EncounterRestController.php

```diff
@@ -37,6 +37,19 @@ public function post($pid, $data)
         return RestControllerHelper::responseHandler($serviceResult, array("eid" => $serviceResult), 200);
     }
 
+    public function put($pid,$eid, $data)
+    {
+        
+        $validationResult = $this->encounterService->validateEncounter($data);
+
+        $validationHandlerResult = RestControllerHelper::validationHandler($validationResult);
+        if (is_array($validationHandlerResult)) {
+            return $validationHandlerResult; }
+ 
```

Comment:
Also, I think to follow the same coding style would be easier to read.

```php
if (is_array($validationHandlerResult)) {
    return $validationHandlerResult;
}
```

Reply:
Hello @stu01509 I agree. This is more readable, but looking at the other methods within the class, thats the format they all follow. Very happy to change it if their is an agreement on style for braces.

Reply:
Hi @prondubuisi 
Let us wait for @bradymiller  sir reply If he agrees,
we will have other jobs to do :)

Reply:
Lol @stu01509. I am currently unemployed. So yes!

Reply:
the psr2 travis failed but once issue #3193 is incorporated in easy dev it'll be caught and fixed :)

Reply:
hi @prondubuisi and @stu01509 , I've recently noted those lines, which agree don't look right (I am surprised psr2/travis never failed them). So feel free to fix them up.

Reply:
i will work on this one

Reply:
I assume you want to add to the FHIR encounter resource. I'll open an issue and assign to you.

Reply:
> I assume you want to add to the FHIR encounter resource. I'll open an issue and assign to you.

Hello @sjpadgett. I think he wants to work this https://github.com/openemr/openemr/issues/3220 . Made a mistake of commenting on the pull request rather than the issue.

Reply:
done @bradymiller